ui/src/app/base/components/ActionForm/ActionForm.tsx

```diff
@@ -91,97 +94,68 @@ const getLabel = (
   }
 };
 
-export type Props<V, E = null> = FormikFormProps<V, E> & {
-  actionDisabled?: boolean;
-  actionName?: string;
-  clearHeaderContent?: (...args: unknown[]) => void;
+export type Props<V, E = null> = Omit<
+  FormikFormProps<V, E>,
+  "buttonsAlign" | "saved" | "saving" | "savingLabel" | "submitLabel"
+> & {
+  actionName: string;
   loaded?: boolean;
   modelName: string;
-  onSuccess?: () => void;
-  processingCount?: number;
-  selectedCount?: number;
+  processingCount: number;
+  selectedCount: number;
 };
 
 const ActionForm = <V, E = null>({
-  actionDisabled,
   actionName,
   buttonsBordered = false,
   children,
-  clearHeaderContent,
   errors,
   loaded = true,
   modelName,
   onSubmit,
-  onSuccess,
   processingCount,
   selectedCount,
   ...formikFormProps
 }: Props<V, E>): JSX.Element | null => {
-  const [processing, setProcessing] = useState(false);
-  const [saved, setSaved] = useState(false);
```

Comment:
This logic is no longer the responsibility of `ActionForm` but of the forms that use it (e.g. `CloneForm`)